scripts/build/setup/termux_setup_meson.sh

```diff
@@ -26,6 +26,12 @@ termux_setup_meson() {
 				79ecf0e16f613396f43621a928df6c17e6260aa190c320e5c01adad94abd07ad
 			patch --silent -p1 -d "$MESON_TMP_FOLDER" < "$MESON_0_61_2_GTKDOC_PATCH_FILE"
 		fi
+		# Patch meson to always return true for libintl check.
+		patch --silent -p1 -d "$MESON_TMP_FOLDER" <"$TERMUX_SCRIPTDIR"/scripts/build/setup/meson_libintl.patch || {
```

Comment:
Nitpick: the rest of this script, and most other script, use spaces on both sides of `=`, `<` and so on

Reply:
`shfmt` default was this style, so I didn't bother to change it. But changed now.